internal/ng_apf_library/ng_apf_library.bzl

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@build_bazel_rules_nodejs//internal/common:providers.bzl", "ScriptsProvider")
```

Comment:
it's only loaded in this one file, so better to just be a private symbol here and not add a new common API file

Reply:
I forgot to load it in `sources_aspect.bzl`, so it's now used in two locations.